### ISSUE TITLE
Ubuntu 20.04 via WSL2 issues

### ISSUE THREAD

**Issue by j0npulsifer** [andremedeiros/loon#37]
:wave:

i tried to `loon up` this repo on Ubuntu 20.04 in WSL2 and ran into some issues.

**Environment:**

```sh
$ uname -rv
4.19.104-microsoft-standard #1 SMP Wed Feb 19 06:37:35 UTC 2020

$ bash --version
GNU bash, version 4.4.23(1)-release (x86_64-unknown-linux-gnu)

$ nix --version; nix-channel --list
nix (Nix) 2.3.6
nixpkgs https://nixos.org/channels/nixpkgs-unstable
```

**Things:**

1. It'd be helpful to automatically determine which interface to add an alias to, in my case it is `lo`. We can probably do this automagically for most cases.

```diff
diff --git a/internal/task/networking.go b/internal/task/networking.go
index 077e335..1500265 100644
--- a/internal/task/networking.go
+++ b/internal/task/networking.go
@@ -39,11 +39,12 @@ func (n *Networking) Check(_ context.Context, p *project.Project) (bool, error)
 func (*Networking) Resolve(_ context.Context, p *project.Project) error {
        return executor.Execute([]string{
                "sudo",
-               "ifconfig",
-               "lo0",
-               "alias",
+               "ip",
+               "addr",
+               "add",
                p.IP.String(),
-               "255.255.255.0",
+               "dev",
+               "lo",
        })

```

2. `loon up` reports it's allllll gooood 😎 but all the shell scripts fail with [Text file busy](https://utcc.utoronto.ca/~cks/space/blog/unix/WhyTextFileBusyError)

```sh
$ rm -rf .loon/

$ loon doctor
You're all good!

$ loon up
✓ Installing software
Ready to go!

$ strace -o strace.txt loon build
bash: /tmp/loon.sh179287157: Text file busy

$ cat /tmp/loon.sh179287157
go generate
go build \
  -ldflags="-X github.com/andremedeiros/loon/cmd.version=$(git rev-parse --short HEAD)"
```

strace output from `loon build`
[strace.txt](https://github.com/andremedeiros/loon/files/4873657/strace.txt)

**Comment by andremedeiros** [comment]
This might be a weird situation with the file not being closed at the right time. I'll 👀

**Comment by andremedeiros** [comment]
This should have been fixed by #38. Let me know if it doesn't work for you!
